apps/web/src/apis/auctions/useGetPersonaTypes.ts

```diff
@@ -0,0 +1,18 @@
+import type { UseQueryOptions } from '@tanstack/react-query';
+import { useQuery } from '@tanstack/react-query';
+
+import { get } from '..';
+
+interface GetPersonaTypesResponse {
+  productTypes: { name: string }[];
+}
+
+const getPersonaTypes = async (): Promise<GetPersonaTypesResponse> => get('/auctions/products/types');
```

Comment:
```suggestion
const getPersonaTypes = () => get<GetPersonaTypesResponse>('/auctions/products/types');
```

- 리턴 타입은 추론되게 하면 어떨까용?
- await 문이 없어서, async도 필요 없을 거 같은데 어떻게 생각하시나용? 문맥상 비동기인 것도 추론 가능할 거 같아서요 🤔